src/calculateRentalCost.js

```
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
```

Potential issue: The function does not handle the case when `days` is 0 or negative. If the requirements or checklist specify input validation, you should add a check to return 0 or throw an error for invalid input.